### DISCUSSION TITLE
单卡3090成功运行 Yi-34B-Chat-AWQ 34B参数模型

### DISCUSSION THREAD

**Discussion by knifecaojia** [chatchat-space/Langchain-Chatchat#2582] [Show and tell]
无聊逛知乎发现这篇文章
https://zhuanlan.zhihu.com/p/669147983
我想试试在3090单卡环境是否能够运行
![image](https://github.com/chatchat-space/Langchain-Chatchat/assets/19563544/ac8acf7f-dacc-4027-8811-be6db241aab1)

**Comment by EricLiuAno** [comment]
能问一下你的python、cuda的版本吗？我这边部署会报很多错。

**Comment by zRzRzRzRzRzRzR** [comment]
可以的，就是载入方式要自己整
